src/hook/useSelectCardBrand.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+import { CARD_BRANDS } from '@/constant';
+import { CardBrand } from '@/type';
+
+export const useSelectCardBrand = () => {
+  const [cardBrand, setCardBrand] = useState<CardBrand>({ label: '', color: '' });
+  const selectCardBrand = (card: CardBrand) => {
+    const selectedCardBrand = CARD_BRANDS.find((cardBrand) => cardBrand.label === card.label);
+    if (selectedCardBrand) {
+      setCardBrand(selectedCardBrand);
+    }
+  };
+  return {
+    cardBrand,
+    selectCardBrand,
+  };
```

Comment:
```suggestion
  return {
    value: cardBrand,
    select: selectCardBrand,
  };
```
저의 경우 위와 같이 내보낸 다음에

```tsx
const cardBrand = useSelectCardBrand();

console.log(cardBrand.value);
cardBrand.select();
```

이런식으로 사용하는 편이랍니다!
내부에서 어떤 이름을 지었든 바깥에서 사용할 때의 모습을 생각하는거죠!

Reply:
> 저의 경우 위와 같이 내보낸 다음에
> 
> ```tsx
> const cardBrand = useSelectCardBrand();
> 
> console.log(cardBrand.value);
> cardBrand.select();
> ```
> 
> 이런식으로 사용하는 편이랍니다! 내부에서 어떤 이름을 지었든 바깥에서 사용할 때의 모습을 생각하는거죠!

일관성을 가질수 있는 좋은 컨벤션이네요! 👀